events-manager/src/main/java/metcamp/backend/eventsmanager/controllers/EventController.java

```diff
@@ -1,34 +1,51 @@
 package metcamp.backend.eventsmanager.controllers;
 
+import metcamp.backend.eventsmanager.entities.model.Event;
+import metcamp.backend.eventsmanager.service.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.util.StringUtils;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.ArrayList;
 import java.util.Map;
 
 @RestController
 @RequestMapping("/metcamp/backend/events")
+
 public class EventController {
 
+    @Autowired
+    private EventService eventService;
+
     @GetMapping
-    public ResponseEntity getAllEvents(){
-        return ResponseEntity.ok(Map.of("events","[GET Lista de eventos]"));
+    public ResponseEntity<Map<String,Object>> getAllEvents(){
+        ArrayList<Event> temporalList = eventService.getAllEvents();
+        return ResponseEntity.ok(Map.of("events",temporalList)); //200
```

Comment:
Podés simplificar esto en una sola línea para no crear un ArrayList que luego no se vuelve a usar:
`return ResponseEntity.ok(Map.of("events",eventService.getAllEvents()));`
Si tenes poquitos eventos puede no hacer una gran diferencia pero a medida que esa lista comienza a crecer, esa variable temporalList se hace más grande y eso tiene un costo a nivel de uso de memoria. Mañana si queres hablamos un ratito sobre garbage collection :)